dts/bindings/cpu/nordic,riscv.yaml

```diff
@@ -6,3 +6,13 @@ description: Nordic Semiconductor RISC-V CPU
 compatible: "nordic,riscv"
 
 include: riscv,cpus.yaml
+
+properties:
+  nordic,bus-width:
```

Comment:
Maybe some comments from the main PR could be applied here first, starting with:

https://github.com/zephyrproject-rtos/zephyr/pull/68043#discussion_r1474175575
https://github.com/zephyrproject-rtos/zephyr/pull/68043#discussion_r1474175903

Reply:
adjusted to `nordic,vpr` per my last comment